### ISSUE TITLE
Build and publish 'Exposure to burn pits and other specific environmental hazards' page in Spanish

### ISSUE THREAD

**Issue by moviedosoco** [department-of-veterans-affairs/va.gov-team#85779]
## Issue description
We need to build this page in Drupal, review it, and publish it.
-https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/translation-work/State%20Department%20translation%20work/disability%20hub/Exposure%20to%20burn%20pits%20and%20other%20specific%20environmental%20hazards.docx

## Tasks
- [ ] Build page in Drupal (Victoria)
- [ ] Review (Sara)
- [ ] Publish (Sara)
- [ ] Send live page to State Dept. for QA (Randi)
- [ ] Input any changes (Victoria)
- [ ] Review (Sara)
- [ ] Publish with Spanish toggle (Sara)

## Acceptance Criteria
- [ ] Page is live 

## Configure this issue
- [ ]  #51139

**Comment by RLHecht** [comment]
@moviedosoco Here's the translation back from State. This page is ready to be built in Drupal.

[SPA-Exposure to burn pits and other specific environmental hazards.docx](https://github.com/user-attachments/files/16234810/SPA-Exposure.to.burn.pits.and.other.specific.environmental.hazards.docx)

**Comment by moviedosoco** [comment]
Hi @RLHecht, this page is built in Drupal. 

HI @saratorres2 The page "Exposición a fosas para la quema de residuos a cielo abierto y otros peligros ambientales específicos" is ready to be reviewed and published:
https://preview-prod.vfs.va.gov/preview?nodeId=73004

**Comment by saratorres2** [comment]
@RLHecht @moviedosoco This page is copy edited and ready for publishing, pending SME approval (1/14).

**Comment by juliana-jara** [comment]
@RLHecht What is the status of this one? Has State reviewed it?

**Comment by RLHecht** [comment]
Yes, State reviewed this. I think we changed some content on the English page not that long ago (either adding ore removing some presumptive conditions). Can you look at revisions on the English page and see if we need to also make changes to the Spanish one? We can send them to State if you think it's warranted. otherwise you can make the changes. Once that's done I think we can publish and can then send the live page back to State for QA. Thanks!

**Comment by juliana-jara** [comment]
Hi @RLHecht . I just compared the live page in English with the one in Spanish that is in production (it is the latest draft). They match and nothing needs to be changed. My only suggestion is to remove the words in the screenshot below as they don't make sense grammatically.

<img width="642" height="200" alt="Image" src="https://github.com/user-attachments/assets/393e7c1a-3a9a-41b6-b0b3-7f5d4910fdc8" />

**Comment by RLHecht** [comment]
Going to move this to icebox. We're pausing work on translation for pages that aren't already live. When we pick this back up, this page is ready to publish once we've sent it to VA partners as an FYI.
